ingest-mongodb-public/src/sources.ts

```diff
@@ -553,6 +569,7 @@ export const sourceConstructors: SourceConstructor[] = [
       snootyDataApiBaseUrl: "https://snooty-data-api.mongodb.com/prod/",
     }),
   () => makeDevCenterDataSource(devCenterProjectConfig),
+  async () => makeMongoDbUniversityDataSource(universityConfig),
```

Comment:
Do we need this `async` here if we're not `await`ing anything?

```suggestion
  () => makeMongoDbUniversityDataSource(universityConfig),
```

Reply:
yeah, it needs to return a promise